pkg/engine/template_generator.go

```diff
@@ -282,10 +282,35 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 		"IsAzureCNI": func() bool {
 			return cs.Properties.OrchestratorProfile.IsAzureCNI()
 		},
+		"IsUsingCosmos": func() bool {
+			return nil != cs.Properties.MasterProfile && helpers.IsTrueBoolPointer(cs.Properties.MasterProfile.UseCosmos)
+		},
+		"strIsUsingCosmos": func() string {
+			if nil != cs.Properties.MasterProfile && helpers.IsTrueBoolPointer(cs.Properties.MasterProfile.UseCosmos) {
+				return "TRUE"
+			}
+			return "FALSE"
+		},
+		"GetCosmosAccountName": func() string {
+			if nil != cs.Properties.MasterProfile {
+				etcdAccountNameFmt := "%sk8s"
+				return fmt.Sprintf(etcdAccountNameFmt, cs.Properties.MasterProfile.DNSPrefix)
+			}
+			return "TEST"
```

Comment:
shouldn't this return empty string?

Reply:
To be honest this was added to allow tests to work. I am not sure what the test expects. I will try with empty string. thank you